tests/saw/Makefile

```diff
@@ -245,6 +245,9 @@ BIKE_R1_COPY = $(addprefix s2n/pq-crypto/bike_r1/, $(notdir $(BIKE_R1_C)))
 BIKE_R2_C = $(wildcard ../../pq-crypto/bike_r2/*.c) $(wildcard ../../pq-crypto/bike_r2/*.h) ../../pq-crypto/bike_r2/Makefile
 BIKE_R2_COPY = $(addprefix s2n/pq-crypto/bike_r2/, $(notdir $(BIKE_R2_C)))
 
+KYBER_R2_C = $(wildcard ../../pq-crypto/kyber_r2/*.c) $(wildcard ../../pq-crypto/kyber_r2/*.h) ../../pq-crypto/kyber_r2/Makefile
+KYBER_R2_COPY = $(addprefix s2n/pq-crypto/kyber_r2/, $(notdir $(SIKE_R2_C)))
```

Comment:
Looks like there's a typo on this like. Should be `KYBER_R2_C` instead of `SIKE_R2_C`.

Reply:
Corrected.